service/resource/legacyv2/resource.go

```diff
@@ -1652,10 +1673,23 @@ func (s *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 
 		_, err = s.awsClients.CloudFormation.CreateStack(&stackInput)
 		if err != nil {
-			return err
+			return microerror.Mask(err)
+		}
+		err = s.awsHostClients.CloudFormation.WaitUntilStackCreateComplete(&cloudformation.DescribeStacksInput{
+			StackName: stackInput.StackName,
+		})
```

Comment:
I see why we need to wait but I have a concern we'll block the operator for a long time. Due to the issue with the framework not processing resources in parallel.

Could we launch the post stack and have it wait for the main stack resources?

Reply:
Indeed this can be a problem, also we currently don't get any feedback about what's going on during any stack creation. I think the host pre and post stack are not a real problem, the creation is fast, but yes, we should do something to address the sequential launch.

Agree to try to resolve it once the resources are split.